src/PatientChecking/PatientChecking/Controllers/AddressesController.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using PatientChecking.Services.ServiceModels;
+
+namespace PatientChecking.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class AddressesController : ControllerBase
+    {
+        private readonly PatientCheckInContext _context;
```

Comment:
Should create a `PatientService` and add `PatientCheckInContext` inside it, then use `IPatientService` interface in this controller. It should be loosely coupled.

Reply:
DI will be applied when we develop each feature.